requirements.txt

```diff
@@ -1 +1,3 @@
 Click
+pyyaml
+neurodocker
```

Comment:
is a requirements.txt file necessary?

Reply:
If we're going to have this (I can't remember the reasoning), I'd prefer it to be synced with `setup.cfg` as in Pydra.

Reply:
removed